services/src/main/java/org/keycloak/services/resources/admin/RealmAdminResource.java

```diff
@@ -416,10 +419,25 @@ public void updateRealmEventsConfig(final RealmEventsConfigRepresentation rep) {
         }
         
         if(dateFrom != null) {
-            query.fromDate(dateFrom);
+            SimpleDateFormat df = new SimpleDateFormat("yyyy-MM-dd");
+            Date from = null;
+            try {
+                from = df.parse(dateFrom);
+            } catch (ParseException e) {
+                e.printStackTrace();
```

Comment:
This should return an error (or throw exception), not just "swallow" the exception